### PR TITLE
Add breadcrumb navigation to docs.

### LINKED ISSUE #38: Consider a breadcrumb nav
As @th-h suggested in #36, a breadcrumb navigation might support orientation within the site, especially the docs. Not sure if that's possible in Jekyll, though. It should be generated automagically/on generation of the site, not hard-coded.

### PR DESCRIPTION
**Pull request by th-h**
Needs CSS form website_assets.

Fixes #38.

Signed-off-by: Thomas Hochstein <thh@inter.net>

### CODE REVIEW

**Review by yellowled — APPROVED** [review]
I don't have the time to check out your changes locally, so I'll just approve, assuming you have tested this. :)

**Comment by th-h** [comment]
Someone (@yellowled) will have to rebuild the CSS file (see s9y/website_assets#4) before this PR can be merged.

**Pull request merged**
